package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -46,11 +46,33 @@ selection parser. The following applies to all selections:
   necessary).
 * Selections are parsed left to right and parentheses can be used for
   grouping.
+* You can use the singular name of any topology attribute as a selection
+  keyword, including a :class:`~MDAnalysis.core.topologyattrs.TopologyAttr`
+  that you have defined yourself, providing that the attribute ``dtype``
+  is one of ``int``, ``float``, ``str`` (or ``object``), or ``bool``.
+  However, the topology must contain this attribute information for
+  the selection to work.
+
+    * Selections of attributes that are integers or floats can use the
+      syntax "myTopologyAttr 0 - 2", "myTopologyAttr 0:2", or
+      "myTopologyAttr 0 to 2", to select a range with
+      both ends inclusive. Whitespace and negative numbers are allowed.
+    * "myTopologyAttr 0" can be used to select all atoms
+      matching the value; however, this can be tricky with floats because of
+      precision differences and we recommend using a range like above when
+      possible.
+    * Boolean selections default to True, so "myTopologyAttr" and
+      "myTopologyAttr True" both give all atoms with
+      ``myTopologyAttr == True``.
 
 
 Simple selections
 -----------------
 
+This is a non-exhaustive list of the available selection keywords. As noted
```

Comment:
Can we generate selection docs on the fly, based on the topology?

How do we document in the static online docs? With asterisk next to anything that might not exist?

Reply:
> Can we generate selection docs on the fly

I could definitely make a nice table for the user guide. Giving examples is one of those cases where flexibly manual documentation is much nicer than anything auto-generated, though. e.g. "resname ALA TYR" vs "resname GenericString" 

> based on the topology?

This is a bit harder. We could list all available attributes in an error message when the user tries to select something that doesn't exist. Alternatively we could try to hack the `select_atoms` docstring but that seems complicated.

> How do we document in the static online docs? With asterisk next to anything that might not exist?

Anything in https://userguide.mdanalysis.org/1.0.0/topology_system.html#format-specific-attributes is format-specific so we could highlight that.